lib/src/base-styles/_nx-forms.scss

```diff
@@ -10,88 +10,83 @@
 
 .nx-form {
   @include container-vertical;
-  border-top: $nx-border-dark;
-  margin: 0;
-  padding-top: 20px;
+  margin: 0 0 $nx-spacing-xl 0;
   position: relative; // to make a containing block for validation popovers
 }
 
-.nx-form--simple {
-  border-top: none;
-  padding: 0;
-}
+.nx-form-footer {
+  @include container-horizontal;
+
+  border-top: 1px solid #e9e9e9;
+  padding: $nx-spacing-l;
+
+  .nx-btn-bar {
+    margin: 0;
+    text-align: right;
```

Comment:
This should be in `.nx-btn-bar`s styles in general (in the other PR I guess)

Reply:
> This should be in `.nx-btn-bar`s styles in general (in the other PR I guess)

I had already moved it here: https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/8d7de5adb14d6437832b48e3f73dcd28f491d9df